src/py/mains/consensus.py

```diff
@@ -68,9 +90,9 @@ def get_alignment(seq1, seq0, edge_tolerance = 1000):
 
 def get_consensus_without_trim( c_input ):
     seqs, seed_id, config = c_input
-    min_cov, K, local_match_count_window, local_match_count_threshold, max_n_read, min_idt, edge_tolerance, trim_size = config
-    if len(seqs) > max_n_read:
-        seqs = seqs[:max_n_read]
```

Comment:
See? Look at the old code at lines 72-73. This was the old meaning of `max_n_read`, and I think we should preserve that.